packages/protocol-kit/src/adapters/web3/Web3Adapter.ts

```diff
@@ -64,6 +64,7 @@ class Web3Adapter implements EthAdapter {
     const balance = defaultBlock
       ? await this.#web3.eth.getBalance(address, defaultBlock)
       : await this.#web3.eth.getBalance(address)
+    // FIXME Web3 Adapter is forced to return an Ethers type
```

Comment:
We should probably create a new ticket for this so we don't forget

Reply:
https://github.com/safe-global/safe-core-sdk/issues/413